packages/StepTracker/Step/Step.jsx

```diff
@@ -34,21 +39,27 @@ const Step = ({ label, status, stepNumber }) => {
 
 Step.propTypes = {
   /**
-   * The label.
+   * The label of this step.
```

Comment:
Step is not a user facing component right? If so, it shouldn't be included in styleguidist and therefore does not need any prop descriptions. Just the propType definitions is fine, no need for the comments.